### ISSUE TITLE
rpminspect doesn't run any inspections on s390x

### ISSUE THREAD

**Issue by sharkcz** [rpminspect/rpminspect#968]
**Describe the bug**
As far as I can tell rpminspect doesn't run any inspections on s390x, see eg. #963 

**To Reproduce**
`rpminspect-redhat -v -k -T ALL 49154443`

**Expected behavior**
inspections being run

**Environment:**
 - Fedora 36 on s390x with
````
rpminspect-data-redhat-1.7-0.1.202211071626git69ef2e6.fc36.noarch
rpminspect-data-fedora-1.10-0.1.202210271518git4429e19.fc36.noarch
rpminspect-1.10-1.fc36.s390x
````

**Comment by dcantrell** [comment]
I can't reproduce this either locally.  Inspections run fine on s390x builds from my x86_64 workstation.  However, I see you are using rpminspect-1.10 as packaged in F36 and I think that might be too old at this point.

I'm about to do the 1.11 release but also you can install it from Copr, which I what I recommend now to at least verify that one works fine.  But today I've been having issues accessing Copr, so maybe there's an outage.  If you are able to install the latest Copr build of rpminspect and run that successfully, let me know.

**Comment by sharkcz** [comment]
The problem is when running the inspections "natively" on s390x machine (Fedora 36 on s390x hw), not for checking s390x builds on x86 machine. I would check with rpminspect 1.11 from COPR, but it suffers from #966.

**Comment by dcantrell** [comment]
If you can give me access to an s390x host, I can work on these problems.

**Comment by sharkcz** [comment]
I have sent you an email with the details.

**Comment by sharkcz** [comment]
I wonder if rpminspect expects to be run from a directory with dist-git checkout for the package ... Because on the reproducer host I see
````
...
debug: curl_helper (383): src=|https://.../tasks/6774/49196774/s390utils-2.24.0-1.el9.s390x.rpm|
dst=|/var/tmp/rpminspect/scratch-49196744.Mw7mF3/after/s390x/s390utils-2.24.0-1.el9.s390x.rpm|
=> s390utils-2.24.0-1.el9.s390x.rpm                                                 [####################################################################################]
debug: main (827): product_release=(null)
````
when run with `-d`

**Comment by dcantrell** [comment]
I have confirmed that this bug exists in the released rpminspect-1.10 version, but the latest Copr build of rpminspect has this bug resolved.  The fix will be in v1.11.
